pkgs/development/libraries/pupnp/default.nix

```diff
@@ -1,6 +1,8 @@
-{ fetchFromGitHub, stdenv, autoreconfHook, pkg-config }:
+{ fetchFromGitHub, stdenv, autoreconfHook, pkg-config
+, enableReuseAddr ? false }:
 
-stdenv.mkDerivation rec {
+with stdenv;
```

Comment:
Please revert this change.

Reply:
sure.
Can you please clarify what's the problem with that writing (if any :)?

Reply:
with should be applied to the smallest scope possible.